app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -420,6 +464,18 @@ private int currentDrawerMenuIcon(final @Nullable User user) {
       }
     }
 
+    private Observable<Response> makeCall(final @NonNull Uri uri) {
```

Comment:
What do you think about renaming this to something else? `apiRequest` or `apiRequestForUri`?

Reply:
it's not a request to an REST API endpoint, you could call any uri, that's why the name is so generic